client/src/helpers/translation.js

```diff
@@ -19,3 +19,20 @@ export const handleTranslation = (
     }
   }
 }
+
+export const getNavigatorLanguage = () => {
+  let language = navigator.language
+  const [formattedLanguage] = language.split('-')
+  if (formattedLanguage !== 'fr' && formattedLanguage !== 'en') {
+    return formattedLanguage === 'en'
+  }
+  return formattedLanguage
+}
+
+export const handleAutoTranslation = (navigatortLanguage, questionLanguage) => {
+  let isAutoTranslated = false
+  if (navigatortLanguage !== questionLanguage) {
+    isAutoTranslated = true
+  }
+  return isAutoTranslated
+}
```

Comment:
Cette fonction ne semble pas être utilisée

```suggestion
export const shouldAutoTranslate = (navigatorLanguage, questionLanguage) => {
  return navigatorLanguage !== questionLanguage
}
```